anodotSubmitter/AnodotSubmitterInterface.go

```diff
@@ -2,6 +2,6 @@ package anodotSubmitter
 
 import "github.com/anodot/anodot-common/anodotParser"
 
-type submitter interface {
-	SubmitMetrics(metrics *[]anodotParser.AnodotMetric)()
```

Comment:
safe to pass slices by values, since it has already a pointer to an array, so new allocation will no happen